source/plugins/projects/projects/index.js

```diff
@@ -1217,7 +1217,7 @@ let projects = {
 	 * 
 	 * @example
 	 * 
-	 * 		saveFile('MyNewProject', 'FileName', [1, 2, 3]);
+	 * 		saveFile(project, 'FileName', [1, 2, 3]);
```

Comment:
As most of the data written in a file will be strings, I suggest having the example with `Buffer.from ('.....')`